creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -218,13 +224,19 @@ module.exports = async function (
         fetchedClockRecords[0] &&
         fetchedClockRecords[0].clock !== localMaxClockVal + 1
       ) {
-        throw new Error(
-          `Cannot sync - imported data is not contiguous. Local max clock val = ${localMaxClockVal} and imported min clock val ${fetchedClockRecords[0].clock}`
-        )
+        return {
+          error: new Error(
+            `Cannot sync - imported data is not contiguous. Local max clock val = ${localMaxClockVal} and imported min clock val ${fetchedClockRecords[0].clock}`
+          ),
+          result: 'failure_sync_secondary_from_primary'
```

Comment:
can you make this result msg more specific eg `failure_import_not_contiguous`
and `failure_import_not_consistent` for below

Reply:
we def want to track these two conditions separately

Reply:
same for the one above